lib/ecmascript.ts

```diff
@@ -5016,6 +5016,12 @@ export function GetOptionsObject<T>(options: T) {
   throw new TypeError(`Options parameter must be an object, not ${options === null ? 'null' : `${typeof options}`}`);
 }
 
+export function CreateOnePropObject<K extends string, V extends unknown>(propName: K, propValue: V) {
```

Comment:
Nit: Can we explicitly write out the return value for this instead of letting it be inferred? Probably something like `{[K]: V}` ?

Reply:
Done in latest commit. The only syntax I could get to work was this: `{ [k in K]: V }`.